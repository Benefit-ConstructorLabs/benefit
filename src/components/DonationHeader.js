import React from 'react';
import PropTypes from 'prop-types';
import '../../styles/components/donation-header.scss';

const DonationHeader = ({ firstName }) => (
  <div className="donation">
    <h2 className="donation__headline">Cashless donations that make a difference.</h2>
    <p className="donation__strapline">Quick. Secure. Cashless.</p>
    <ol className="donation__steps">
      <li className="donation__step">
        Decide how much you would like to donate to
        {firstName}
        .
      </li>
      <li className="donation__step">Set your payment amount.</li>
      <li className="donation__step">
        Help
        {firstName}
        out.
      </li>
    </ol>
  </div>
);

DonationHeader.propTypes = {
  firstName: PropTypes.string.isRequired,
};

export default DonationHeader;
