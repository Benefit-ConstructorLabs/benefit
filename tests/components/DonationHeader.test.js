import React from 'react'
import DonationHeader from '../../src/components/DonationHeader';
import renderer from 'react-test-renderer';

describe('DonationHeader', () => {
  test('should render correctly', () => {
    const tree = renderer
      .create(<DonationHeader />)
      .toJSON()
    expect(tree).toMatchSnapshot();
  });
})