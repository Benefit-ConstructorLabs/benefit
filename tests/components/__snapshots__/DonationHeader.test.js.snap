// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DonationHeader should render correctly 1`] = `
Array [
  <h2
    className="donation__headline"
  >
    Cashless donations that make a difference.
  </h2>,
  <h3
    className="donation__strapline"
  >
    Quick. Secure. Cashless.
  </h3>,
  <ol
    className="donation__steps"
  >
    <li
      className="donation__step"
    >
      Decide how much you would like to donate to Bill.
    </li>
    <li
      className="donation__step"
    >
      Set your payment amount.
    </li>
    <li
      className="donation__step"
    >
      Help Bill out.
    </li>
  </ol>,
]
`;
